pkgs/applications/version-management/git-pw/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, git
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "git-pw";
+  version = "2.5.0";
+
+  PBR_VERSION = version;
+
+  src = fetchFromGitHub {
+    owner = "getpatchwork";
+    repo = "git-pw";
+    rev = version;
+    hash = "sha256-MUothqfdDALKpHzFy5O67vAfbyZOa11YuWRsvpwrSPY=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pyyaml
+    arrow
+    click
+    requests
+    tabulate
+    pbr
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [
+    pytest
+    pytest-cov
```

Comment:
Please patch the coverage parts out as those don't add value for us. E.g., `--cov=git_pw --cov-report`.